src/views/HeaderView.js

```diff
@@ -0,0 +1,52 @@
+import routes from '../router';
+import { qs, on, qsAll, emit, navigate } from '../helpers';
+import '../styles/header.scss';
+
+export default class HeaderView {
+  constructor($body) {
+    this.$body = $body;
+    this.template = new Template();
+    this.bindEvents();
+  }
+
+  render() {
+    this.$body.innerHTML = this.template.getHeader();
+  }
+
+  async bindEvents() {
+    const header = await this.template;
+    header && on(qs('ul'), 'click', event => this.handleNavigate(event));
```

Comment:
on, qs같은 헬퍼함수를 만들어낼 필요가 없어보여요.

```suggestion
    const $headerIcons = document.querySelector('ul')
    $headerIcons.addEventListener('click', event => this.handleNavigate(event)));
```

Reply:
해당 헬퍼함수들 삭제해 준 후, addEventListener, querySelector를 사용하도록 수정했습니다!